src/main/java/com/solvd/micro9/users/web/WebConfig.java

```diff
@@ -43,4 +54,69 @@ public WebClient.Builder webClient() {
         return WebClient.builder();
     }
 
+    public GraphQLScalarType localDateTimeScalar() {
+        return GraphQLScalarType.newScalar()
+                .name("LocalDateTime")
+                .description("Java 8 LocalDateTime as scalar")
+                .coercing(new Coercing<LocalDateTime, String>() {
+                    @Override
+                    public String serialize(final Object input) {
+                        if (input instanceof LocalDateTime) {
+                            DateTimeFormatter formatter =
+                                    DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+                            return ((LocalDateTime) input).format(formatter);
+                        } else {
+                            throw new CoercingSerializeException(
+                                    "Expected a LocalDateTime object"
+                            );
+                        }
+                    }
+
+                    @Override
+                    public LocalDateTime parseValue(final Object input) {
+                        try {
+                            if (input instanceof String) {
+                                return LocalDateTime.parse((String) input);
+                            } else {
+                                throw new CoercingParseValueException("Expected a String");
+                            }
+                        } catch (DateTimeParseException e) {
+                            throw new CoercingParseValueException(
+                                    String.format("Not a valid date: '%s'.", input), e
+                            );
+                        }
+                    }
+
+                    @Override
+                    public LocalDateTime parseLiteral(final Object input) {
+                        if (input instanceof StringValue) {
+                            try {
+                                return LocalDateTime.parse(((StringValue) input)
+                                        .getValue());
+                            } catch (DateTimeParseException e) {
+                                throw new CoercingParseLiteralException(e);
+                            }
+                        } else {
+                            throw new CoercingParseLiteralException(
+                                    "Expected a StringValue"
+                            );
+                        }
+                    }
+                }).build();
+    }
+
+    public ValidationSchemaWiring schemaWiring() {
```

Comment:
@eyakauleva as far I understand this ValidationSchemaWiring also should be represented as Spring Bean

Reply:
@h1alexbel I call this method in another method, then why it should be a bean?

Reply:
@eyakauleva for code reuse?

Reply:
@h1alexbel but I call this method only once

Reply:
@eyakauleva I know, but what about future usages?

Reply:
@h1alexbel according to YAGNI principle features should only be added when required

Reply:
@eyakauleva YAGNI is about features or [FR](https://www.wikiwand.com/en/Functional_requirement), not about maintainability, which is [NFR](https://www.wikiwand.com/en/Non-functional_requirement)

Reply:
done 679c337